easybuild/easyconfigs/s/Sambamba/Sambamba-0.7.1.eb

```diff
@@ -0,0 +1,36 @@
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+#
+# Author: Alex Domingo
+# SISC - Vrije Universiteit Brussel
```

Comment:
@lexming I know this is perhaps nitpicking, but you may want to specify that recent versions should be credited to you?

Reply:
Good idea, fixed :+1: